cni/test/e2e-taint/testdata/podsConfig/addon-test2-in-test2.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: addon-in-test2
+  namespace: test2
+  labels:
+    addon: test2
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+        addon: test2
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.14.2
```

Comment:
if we are using test pods they must be the echo app: https://github.com/istio/istio/blob/de59ea74bc123cdd32901ddc1b0435679e00a0c4/pkg/test/framework/components/echo/kube/deployment.go#L66 not pulling random images

Reply:
But we should probably discuss what the goals of the test are before we dig into this

Reply:
containers are not important,  but it requires to be a daemonset